frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -214,7 +214,7 @@ const SECTIONS = updateSectionsWithPlugins({
         type: "string",
         required: false,
         widget: SettingCommaDelimitedInput,
-        validations: [["email_list", t`That's not a valid email addresses`]],
+        validations: [["email_list", t`That's not a valid email address`]],
```

Comment:
Do we only allow 1 reply-to email? The setting description tells otherwise and that's a bit confusing.
![image](https://user-images.githubusercontent.com/1937582/178906703-3717cbef-8d2e-4ef0-be4c-6a6dace6214f.png).

And you might want to fix the copy here as well?
https://github.com/metabase/metabase/blob/68572a75df81e5a7e51e5f87fc8b5b89a5c9175c/frontend/src/metabase/admin/settings/components/SettingsBatchForm.jsx#L24

Reply:
If we support multiple reply-to addresses, then we're gonna change the FE part a bit too. Now the PR seems to assume we can only input 1 email.

Reply:
Good catch. The FE copy was wrong and I've since updated it. We currently support one reply-to email, with the option to extend it to more than one email address in the future, which is why it's stored as a JSON list.